docs/user/config/tsConfig.md

```diff
@@ -89,7 +89,7 @@ module.exports = {
 
 #### Disable auto-lookup
 
-By default `ts-jest` will try to find the `tsconfig.json` in your project. But you may want to not use it at all and keep TypeScript default options. You can achieve this by setting `tsConfig` to `false`.
+By default `ts-jest` will try to find a `tsconfig.json` in your project. But you may not want to use it at all and keep TypeScript default options. You can achieve this by setting `tsConfig` to `false`.
```

Comment:
`tsConfig` to `false` -> `tsConfig` (alias `tsconfig`) to `false`.

Reply:
Do we really want to call out the alias every time `tsConfig` is mentioned?

Reply:
I think maybe not, it's not so mandatory for me to mention it every time. Just a suggestion

Reply:
Just trying not to be too aggressive with the reminder. I think it’s mentioned in the places where it should be.